docs/user/tutorials/vote-extensions/mitigate-frontrunning.md

```diff
@@ -0,0 +1,500 @@
+# Mitigate Auction Front-Running
+
+In this tutorial, we will be covering:
+
+* [What are Vote extensions?](#what-are-vote-extensions)
+* [Context](#context)
+* [Simulation of Auction Front-Running](#simulation-of-auction-front-running)
+* [Mitigating Front-running with Vote Extensions](#mitigating-front-running-with-vote-extensions)
+* [Demo of Mitigating Front-Running with Vote Extensions](#demo-of-mitigating-front-running-with-vote-extensions)
+
+## What are Vote extensions?
+
+Vote extensions is arbitrary information which can be inserted into a block. This feature is part of ABCI 2.0, which is available for use in the SDK 0.50 release and part of the 0.38 CometBFT release. 
+
+More information about vote extensions can be seen [here](../../../build/abci/03-vote-extensions.md).
+
+## Context
+
+For this tutorial we are using an example of an application that is mitigating auction front-running. 
+
+Front-running is a potential issue in blockchain systems where a participant gains an advantage by being able to perform actions or transactions ahead of others. In the context of this application, front-running can occur when a validator, who is also a proposer, spots a bid in the mempool and replaces it with their own higher bid. This is also known as "bid sniping".
+
+Please checkout https://github.com/fatal-fruit/abci-workshop to see the example.
```

Comment:
can we get this migrated to the tutorial repo? we usually wouldnt allow this sort of linking for something we will end up hosting in the tutorial repo get merged as it will cause confusion later on. 

we should also mention that the tutorial is in that repo and this walkthrough is assuming you have gone through the tutorial.